System/GraphicalPrimitive.cpp

```diff
@@ -31,13 +31,43 @@ namespace RTE {
 		drawRightPos.m_Y -= targetPos.m_Y;
 	}
 
+/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+	static void lineWithThickness(BITMAP *drawScreen, const Vector &startPos, const Vector &endPos, unsigned char color, const int &thickness) {
+		int x_delta = startPos.GetFloorIntX() - endPos.GetFloorIntX(),
+			y_delta = startPos.GetFloorIntY() - endPos.GetFloorIntY();
+		Vector n(-y_delta, x_delta);
```

Comment:
Please try to figure out a more descriptive name than `n` for this Vector. Also, I think you don't need `x_delta` or `y_delta` at all, since they're just used to construct the Vector.